app/main.py

```diff
@@ -16,4 +18,21 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+
+sorted_variables = {"mutable": [], "immutable": []}
```

Comment:
The `sorted_variables` dictionary is a global variable. Modifying global variables inside functions can lead to unexpected behavior and make the code harder to debug. It would be better to make this dictionary a local variable inside the `sort_variables()` function and return it at the end of the function.